src/objective-c/generated_libraries/RemoteTestClient/RemoteTest.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.prepare_command = <<-CMD
     BINDIR=../../../../bins/$CONFIG
     PROTOC=$BINDIR/protobuf/protoc
```

Comment:
I was unsure whether to change this one. There is no `protoc` binary at that path for me. I couldn't find a built `protoc` binary after running `make` in the root of the project or in the `third_party/protobuf` directory.